.github/workflows/publish-to-npm.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish release to npm
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Publish to npm
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+
+      - name: Setup Node
+        uses: actions/setup-node@v4.1.0
```

Comment:
Don't want to cache so using these actions rather than our composite workflows which include caching.